woodwork/utils.py

```diff
@@ -267,6 +267,9 @@ def _to_latlong_float(val):
 def _is_valid_latlong_series(series):
     '''Returns True if all elements in the series contain properly formatted LatLong values,
     otherwise returns False'''
+    dd = import_or_none('dask.dataframe')
+    if dd and isinstance(series, dd.Series):
+        series = series.compute()
```

Comment:
Not sure if we want to pull these values into memory to check them, or if we would be better off to skip this validation for Dask? It can always be addressed in a future issue if this is problematic.

Reply:
Let's create an issue to revisit this later on. We can leave compute as is.

Reply:
Created #627 for this and added it to the Accessor Improvements epic